server/server.go

```diff
@@ -294,6 +306,31 @@ func ReadCommitmentMode(r *http.Request) (commitments.CommitmentMode, error) {
 	return commitments.OptimismAltDA, nil
 }
 
+func ReadCommitmentVersion(r *http.Request, mode commitments.CommitmentMode) (byte, error) {
```

Comment:
I don't think cert version is a part of request. From what I understand, the request body only contains the user's data. After the dispersal is returned from EigenDA (after Put), the call returns a DA cert,

https://github.com/Layr-Labs/eigenda-proxy/blob/main/server/server.go#L214 (commitment is actually the DA cert)

then it prepend meta information in front of the DA cert. https://github.com/Layr-Labs/eigenda-proxy/blob/main/commitments/mode.go#L65

then DA cert version is actually hard coded for now. https://github.com/Layr-Labs/eigenda-proxy/blob/main/commitments/da_service_op.go#L41

Reply:
The cert version I'm getting is the `version byte` described in the docs here: https://github.com/Layr-Labs/eigenda-proxy?tab=readme-ov-file#commitment-schemas 
I understand that the version is currently hard coded, but was thinking this could support future changes when there's more DA types.